resources/views/pages/partials/submit-modal.blade.php

```diff
@@ -15,13 +15,13 @@ class="fixed z-30 inset-0 overflow-y-auto"
          x-transition:leave-start="opacity-100"
          x-transition:leave-end="opacity-0 scale-95"
     >
-        <div class="fixed inset-0 backdrop-filter backdrop-blur-lg backdrop-brightness-75 transition" aria-hidden="true"></div>
+        <div class="fixed inset-0 backdrop-blur-lg backdrop-brightness-75 transition" aria-hidden="true"></div>
 
         <!-- This element is to trick the browser into centering the modal contents. -->
         <span class="hidden sm:inline-block sm:align-middle sm:h-screen" aria-hidden="true">&#8203;</span>
 
         <!-- Modal Content -->
-        <div class="inline-block align-bottom bg-white rounded-lg p-6 sm:p-12 text-left overflow-hidden shadow-xl transform transition-all sm:my-8 sm:align-middle sm:max-w-lg sm:w-full"
+        <div class="inline-block align-bottom bg-white rounded-lg p-6 sm:p-12 text-left overflow-hidden shadow-xl transition-all sm:my-8 sm:align-middle sm:max-w-lg sm:w-full"
```

Comment:
If you add the `relative` class to this div the modal should be above the backdrop.

Reply:
That worked indeed :-) Thanks a lot.